### PR TITLE
Fix/static checks

### PR DESCRIPTION
**Pull request by bignastybryce**
This PR fixes the trivial staticcheck errors.  The remaining ones are for unused code - I will address those as I move that code into packages.

### CODE REVIEW

File: compare_test.go [hunk 1]
```diff
@@ -97,45 +97,18 @@ type TestStruct struct {
...
+	}
+	a := []TestStruct{s}
 	bytes, err := json.Marshal(a)
 	if err != nil {
 		fmt.Println("e", err)
 	}
 	fmt.Println("s", string(bytes))
 }
 
-func aTestJSONDeserialize(t *testing.T) {
```
Comment (edwardbartholomew) [inline compare_test.go:109]: Looks like Jon was testing this from his Makefile at one point.

Comment (bignastybryce) [reply compare_test.go:109]: Do you want to keep it, then?  We could move it into cli_test.go, where the integration tests are.  That way, we could remove the "a" from the front, and it won't show up as unused.

Comment (edwardbartholomew) [reply compare_test.go:109]: after looking/thinking a little more, let's delete it. 👍

**Review by edwardbartholomew — APPROVED** [review]
Looks good

**Pull request merged**